components/Work.js

```diff
@@ -58,6 +58,11 @@ const Work = ({ work, width = '100vw' }) => {
     'fields.hoveredStateTheme',
     'Dark'
   ).toLowerCase();
+  const disableOverlayOnDarkTheme = get(
+    work,
+    'fields.disableOverlayOnDarkTheme',
```

Comment:
Please make this `Boolean` type in Contentful as it is the better fitting data type.